content/api/commands/session.md

```diff
@@ -340,13 +265,27 @@ it('should transfer money between users', () => {
 
 ### Validating the session
 
-If the `validate` function return `false`, throws an exception, returns a
-Promise that resolves to `false` or rejects, or contains any failing Cypress
-command, the session will be considered invalid, and `setup` will be re-run.
+The `validate` function has a slightly different function depending on whether
```

Comment:
> has a slightly different function

I'd re-word to indicate this function should be able to run after setup or a restored context and should not rely on the previous page state. 

It would also be worth expanding on this is intended to ensure authentication with the back-end to ensure the application is authenticated correctly as the user interacts with the page and/or when new requests are made